segpy/ibm_float_packer.py

```diff
@@ -72,11 +72,10 @@ def unpack(self, data, num_items):
 
 
 def _active_packer():
-    name = 'python'
-    if 'cpp' in _EXTENSION_MANAGER and not force_python_ibm_floats:
-        name = 'cpp'
+    if force_python_ibm_floats or 'cpp' not in _EXTENSION_MANAGER:
+        return Packer()
```

Comment:
It seems like we should be able to avoid instantiating a new `Packer` on every call to `_active_packer()`, we call this method a lot, so it would be good to keep the allocation overhead down. The simplest change would be to instantiate always instantiate a Packer instance in global scope and return that every time.

Reply:
Not directly related to this change, but Packer is stateless, so these could be staticmethods with the class just used as a namespace. I guess that would require updating the C++ packer too.